### ISSUE TITLE
Exporting instance domain blocks doesn't export private comment

### ISSUE THREAD

**Issue by Ember-ruby** [glitch-soc/mastodon#2401]
### Steps to reproduce the problem

1. go to /admin/export_domain_blocks/export.csv
2. let it download
3. the downloaded file is missing private comments

### Expected behaviour

private comments should either be included, or there should be an option to specifically export with private comments

### Actual behaviour

downloaded csv lacks private comments

### Technical details

relevant line
https://github.com/glitch-soc/mastodon/blob/main/app/controllers/admin/export_domain_blocks_controller.rb#L72

**Comment by Plastikmensch** [comment]
I don't think that's a bug.
When importing domain blocks from a CSV file, the private comment will be overwritten with `Imported from <file> on <timestamp>`, so including private comments in exports could be confusing or give the wrong impression these would be applied when imported.

I can still see exporting private comments being useful in some cases, but as the name suggests these are supposed to be private and the feature is meant for exporting and importing domain blocks.

**Comment by Ember-ruby** [comment]
> I don't think that's a bug. When importing domain blocks from a CSV file, the private comment will be overwritten with `Imported from <file> on <timestamp>`, so including private comments in exports could be confusing or give the wrong impression these would be applied when imported.

yeah, i get that, that's why i mentioned an option for exporting with private comments

> I can still see exporting private comments being useful in some cases, but as the name suggests these are supposed to be private and the feature is meant for exporting and importing domain blocks.

my particular use case is locally editing/viewing the comments, checking if they're correct, updating them, etc (in which case being able to re-import with the updated private comments would be nice) without using the webui, which is *really* not good for editing a lot of items

**Comment by Ember-ruby** [comment]
i mean, it's an admin option, it should have admin settings
